comment/api/serializers.py

```diff
@@ -22,10 +24,14 @@ def get_profile_model():
 def get_user_fields():
     user_model = get_user_model()
     fields = user_model._meta.get_fields()
+    api_fields = settings.COMMENT_USER_API_FIELDS
+    api_fields.append('profile')
+    # remove duplicates while preseving order
+    api_fields = list(OrderedDict((x, 1) for x in api_fields))
```

Comment:
This probably seems to be a little cleaner and lesser complicated to me.
```suggestion
    api_fields = list(OrderedDict.fromkeys(api_fields))
```